frontend/src/main/components/OurTable.js

```diff
@@ -57,9 +57,12 @@ export default function OurTable({ columns, data, testid = "testid", ...rest })
           return (
             <tr {...row.getRowProps()}>
               {row.cells.map((cell, _index) => {
+                const cellStyle = { textAlign: 'right' };
```

Comment:
This should not be done at the OurTable level, and doesn't seem like it's part of this PR in any case.

You may want to revert this change.